samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -152,6 +152,12 @@ static const char progress_message[] =
 // @brief message for performance counters option
 static const char pc_message[] = "Optional. Report performance counters.";
 
+// @brief message for sorted performance counters option
+static const char pc_sort_message[] = "Optional. Report performance counters and analysis the sort hotpoint opts. "
+                                      " \"sort\" Analysis opts time cost, print by hotpoint order "
+                                      " \"no_sort\" Analysis opts time cost, print by normal order "
+                                      " \"simple_sort\" Analysis opts time cost, only print EXECUTED opts by normal order";
```

Comment:
Could we extend -pc for sorted performance counters?

1. Keep current behavior as default when there is no additional parameter 
2. Do sorted performance counters when "sort" and "simple_sort" are provided.